sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/context/AzureDefaultTokenCredentialAutoConfiguration.java

```diff
@@ -34,13 +38,18 @@ public AzureDefaultTokenCredentialAutoConfiguration(AzureGlobalProperties azureG
     @Bean(name = DEFAULT_TOKEN_CREDENTIAL_BEAN_NAME)
     @ConditionalOnMissingBean(name = DEFAULT_TOKEN_CREDENTIAL_BEAN_NAME)
     @Order
-    public TokenCredential azureTokenCredential(AzureCredentialBuilderFactory<DefaultAzureCredentialBuilder> factory) {
+    public TokenCredential springDefaultAzureCredential(DefaultAzureCredentialBuilderFactory factory) {
```

Comment:
can this take a _AbstractAzureCredentialBuilderFactory_ instead of the default one?

Reply:
Yes.